xml/admin_ceph_datamgm.xml

```diff
@@ -727,13 +738,34 @@ maps appear equivalent
         10
        </para>
       </entry>
+      <entry>
+       <para>
+        region
```

Comment:
Region is no longer used and deprecated in the cli tools as well wrt RGW, the term used is Zonegroups which are a collection of Zones. For clusers older than SES3 this corresponds to Regions which was the older term

Reply:
> Region is no longer used and deprecated in the cli tools as well wrt RGW,

Could you explain what "as well wrt RGW" means, please? Because it is so terse I do not follow. Does it mean that the term "RGW" is deprecated? 

> the term used is Zonegroups which are a collection of Zones. For clusers older than SES3 this corresponds to Regions which was the older term

These are the SES 6 docs, currently. That is all we are considering right now in this change.

Reply:
So regions were the old term for a collection of Zones, since this wasn't exactly mapping the aws concept of a region, these were renamed to Zonegroups. the radosgw admin cli tools and everything else also don't use the region term anymore (https://docs.ceph.com/docs/master/radosgw/multisite/ has a couple of words on this)

Reply:
Please bear in mind that this is solely relating to CRUSH map buckets. I do not work with Ceph, AWS or anything related to it myself, I just help maintain the docs, so I cannot really follow explanations in such terms.

I have reworded it thus:

«
10 | zonegroup | A group of zones. Replaces the older geographically-based "region".
»

Is that better?

Reply:
hm I actually misread this section, I assumed this was a general appendix glossary, purely in crush context I don;t think zones even exist, region could just be a geographical region containing datacenters